### DISCUSSION TITLE
Can we use apks as safely as webpages?

### DISCUSSION THREAD

**Discussion by jayadevanraja** [wasmerio/wasmer#1564] [General]
Can Wasmer (in the future) sandbox Anbox so that we can run arbitrary apk files from the internet, safely and securely, just like browsers sandbox web-pages so that we run untrusted javascript from arbitrary web pages today?

Today when we use the GNU/Linux (GNU Operating System with Linux kernel), we can use Anbox, but running untrusted code is a safety and security risk. If Wasmer can sandbox Anbox, we can run the awesome apps of Android inside our laptops.

**Comment by HaileyFoxInSA** [comment]
Seccomp BPF (already in the GNU/Linux kernel) is the answer you're looking for.

**Reply by jayadevanraja** [reply]
@HaileyFoxInSA Thanks for the answer. But, is that usable as a product (as readily as using a browser)?
